extensions/wikia/UserLogin/UserLoginHooksHelper.class.php

```diff
@@ -198,5 +198,23 @@ static public function onWikiaMobileAssetsPackages( Array &$jsStaticPackages, Ar
 
 		return true;
 	}
+
+	/**
+	 * Hook introducing additional control over account creation
+	 * Currently forbids to create an account if username contains
+	 * circled latin characters, e.g. Ⓐ or ⓜ
+	 * @param User $user
+	 * @param String $message
+	 * @return bool
+	 */
+	static public function onAbortNewAccount( $user, &$message ) {
+		$username = $user->getName();
+		$forbiddenCharacters = '/[\x{24B6}-\x{24E9}]|[\x{1F150}-\x{1F169}]/u';
```

Comment:
Is it possible to change variable name to something that tells me that it's regex pattern?